app/uk/gov/hmrc/agentassurance/controllers/MaaController.scala

```diff
@@ -45,13 +42,13 @@ class MaaController @Inject()(repository: PropertiesRepository,
       .validate[Value].getOrElse(throw new BadRequestException("json failed validation"))
 
     Utr.isValid(newValue.value) match {
-      case true => {
+      case true =>
```

Comment:
pattern matching on booleans is overkill,  if you can use if check, please do otherwise no issues